drivers/display/display_nrf_led_matrix.c

```diff
@@ -85,6 +85,14 @@ DT_FOREACH_PROP_ELEM(MATRIX_NODE, pixel_mapping, CHECK_PIXEL)
 #define ITERATION_COUNT  PIXEL_COUNT
 #endif
 
+#if !USE_PWM
+#define GPIO_NODE(pin)      DT_GPIO_CTLR(MATRIX_NODE, pin)
+#define GPIOTE_PHANDLE(pin) DT_PHANDLE(GPIO_NODE(pin), gpiote_instance)
+#define GPIOTE_IDX(pin)     DT_PROP(GPIOTE_PHANDLE(pin), instance)
+
+static nrfx_gpiote_t gpiote = NRFX_GPIOTE_INSTANCE(GPIOTE_IDX(col_gpios));
```

Comment:
This should not assume a single GPIOTE instance. I think that a similar scheme as in `pwm_nrf_sw.c` should be used instead.

Reply:
Done